### PR TITLE
fix: adds missing Point in maps

### PR DESCRIPTION
**Pull request by cfrancisco**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Previously, in map visualization, when real-time and tracking are enabled, were been lost some points, this PR fixes it. 
PS: The first commit is the solution itself. The following commits fix the lint issues.

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
This PR is connected to dojot/dojot#974;

* **Other information**:

### CODE REVIEW

**Comment by mprevide** [comment]
Merge into bugfix/tracking or something like that. :)

**Comment by mprevide** [comment]
> Merge into bugfix/tracking or something like that. :)

^^

File: src/js/stores/MeasureStore.js [hunk 1]
```diff
@@ -10,20 +10,26 @@ class MeasureStore {
         this.data = {};
         this.tracking = {};
         this.error = null;
+        //this.measureReload = true;
```
Comment (mprevide) [inline src/js/stores/MeasureStore.js]: Remove that :)

File: src/js/views/devices/DeviceMap.js [hunk 2]
```diff
@@ -75,6 +76,10 @@ class DeviceMap extends Component {
         return count;
     }
 
+    // shouldComponentUpdate(nextProps, nextState) {
```
Comment (mprevide) [inline src/js/views/devices/DeviceMap.js]: Remove that

File: src/js/views/utils/Maps.js [hunk 3]
```diff
@@ -51,361 +48,285 @@ const Esri_WorldImagery = L.tileLayer(
...
-        rsi(data);
-      });
-
-      device_list_socket.on("error", data => {
-        // console.log("socket error", data);
-        if (device_list_socket !== null) device_list_socket.close();
-        // getWsToken();
-      });
+    componentDidMount() {
+    // console.log("MapSocket: componentDidMount:");
```
Comment (mprevide) [inline src/js/views/utils/Maps.js]: Remove :)

File: src/js/views/utils/Maps.js [hunk 4]
```diff
@@ -418,138 +339,99 @@ class SmallPositionRenderer extends Component {
...
+                        active_tracking: tmp.active_tracking,
+                        allow_tracking: device.allow_tracking,
+                        key: tmp.unique_key
+                            ? tmp.unique_key
+                            : tmp.id,
+                    });
                 }
             }
         }
         // console.log("parsedEntries (static and dynamics):", parsedEntries);
```
Comment (mprevide) [inline src/js/views/utils/Maps.js]: Remove

File: src/js/views/utils/maps/ContextMenuComponent.js [hunk 5]
```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import PropTypes from 'prop-types';
+
+
+class ContextMenu {
+    // context menu based at: https://codepen.io/devhamsters/pen/yMProm
+    updateCurrentContextMenu(event, root) {
+        // console.log("updateCurrentContextMenu");
```
Comment (mprevide) [inline src/js/views/utils/maps/ContextMenuComponent.js]: Remove

**Comment by cfrancisco** [comment]
> > Merge into bugfix/tracking or something like that. :)
> 
> I think It's not necessary this not a big fix

In fact, it's a bugfix, following the issue reported to it and the fix itself. 
Using the git flow idea, features should be sent to a specific branch and bugfixes to the development branch.

**Comment by cfrancisco** [comment]
Thanks for the comments. About the old commented code, it should be removed in the second moment in all related files. Thus, was created an issue related to it (dojot/dojot#1021).

**Pull request merged**